bedrock/thunderbird/templates/thunderbird/index.html

```diff
@@ -15,9 +15,7 @@
   <h2>{{ high_res_img('thunderbird/landing/wordmark.png', {'alt': _('Thunderbird'), 'width': '304', 'height': '84'}) }}</h2>
 {% endblock %}
 
-{% block site_css %}{% endblock %}
-
-{% block extrahead %}
+{% block page_css %}
```

Comment:
I did think about this one. It's not a browser, and the download button is still visible and functional as-is on IE 7, so I figured it was okay to send fancy CSS only modern browsers. What do you think?